src/xma/src/xmaplugin/xma_session.cpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (C) 2019 Xilinx, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+#include "lib/xma_session.hpp"
+#include "app/xmaerror.h"
+
+namespace xma_core {
+namespace plg {
+
+session::session(int s_id, XmaSessionType s_type, int32_t c_id)
```

Comment:
Prefer initialization to assignment:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#Rc-initialize

Reply:
Agree. Done